drivers/io/io_fip.c

```diff
@@ -176,19 +230,23 @@ static int fip_file_open(io_dev_info_t *dev_info, const uintptr_t spec,
 	int result;
 	uintptr_t backend_handle;
 	const io_uuid_spec_t *uuid_spec = (io_uuid_spec_t *)spec;
+	file_state_t *current_file;
 	size_t bytes_read;
 	int found_file = 0;
 
 	assert(uuid_spec != NULL);
 	assert(entity != NULL);
+	assert((dev_info->info != (uintptr_t)NULL));
+
+	current_file = (file_state_t *)dev_info->info;
 
 	/* Can only have one file open at a time for the moment. We need to
```

Comment:
Does this comment hold anymore?

Reply:
Sure, I will change the comment to "one file per device"